packages/firebase_core/firebase_core/lib/src/firebase.dart

```diff
@@ -63,8 +65,9 @@ class Firebase {
   }
 
   @override
-  int get hashCode => this.toString().hashCode;
+  int get hashCode => toString().hashCode;
 
+  // TODO(rrousselGit): include app name/options
```

Comment:
Probably left overs from our discussion; this one isn't needed, was only for FirebaseApp not Firebase